invenio_accounts/templates/semantic-ui/invenio_accounts/_macros.html

```diff
@@ -0,0 +1,43 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2015-2018 CERN.
```

Comment:
Very minor!
```suggestion
  Copyright (C) 2015-2020 CERN.
```
in all of them

Reply:
I've merged and pushed fixes.